themes/default/content/docs/support/troubleshooting.md

```diff
@@ -105,9 +96,39 @@ To view a trace locally navigate to the [Jaeger UI](http://localhost:16686/searc
 
 ## Common Problems
 
-This section covers a few problems that can arise when working with Pulumi.
+### 403 error fetching plugin
+
+This error commonly occurs when using an arm64 based processor while using an older version of a provider that does not support arm64. It's not possible to upgrade the packages as your state will still be locked to the old version of the provider.
+
+The method for fiing this depends on whether you are using an Intel based computer.
+
+#### Intel based computer
+
+1. Open your Pulumi program on a non-arm64 based computer.
+1. Update your packages (pip / nuget / npm / go) and run `pulumi up`.
+1. Once the update is complete, open the new, updated Pulumi program on your arm64-based system.
+
+#### Non-Intel based computer
+
+1. Remove Pulumi - if you're using Homebrew, `brew remove pulumi` or `rm -rf ~/.pulumi`
+1. Download [latest version of Pulumi]({{< relref "/docs/get-started/install/versions" >}}).
+1. Add Pulumi to path: `export PATH=$PATH:~/.pulumi/bin`
+1. Update packages in your Pulumi program to latest version (for example `npm install @pulumi/aws@latest)
+1. Install Pulumi provider: `arch -x86_64 pulumi plugin install resource {provider_name} v{version}` (where  {provider_name} is the name of the provider, i.e. aws and {version} is the same version number that your package has updated to). `arch` is used to run the selected architecture of a binary, in this case so that you can run the non-ARM64 version of Pulumi on your laptop.
+1. [Login to Pulumi]({{< relref "/docs/intro/concepts/state/#logging-in" >}}).
+1. Run a Pulumi preview: `arch -x86_64 pulumi pre`.
+1. Remove Pulumi again `rm -rf ~/.pulumi`.
+1. [Re-install Pulumi]({{< relref "/docs/get-started/install" >}})
+1. [Login to Pulumi]({{< relref "/docs/intro/concepts/state/#logging-in" >}}).
+1. Run a Pulumi preview to check everything is ok: `pulumi pre`
+
+### 409 Conflict: Another update is currently in progress. {#conflict}
+
+Run `pulumi cancel` to cancel the update.
 
-### [409] Conflict: Another update is currently in progress. {#conflict}
+{{% notes type="warning" %}}
+Warning! If you cancel another person's update, their update will fail immediately.
+{{% /notes %}}
 
 One of the services that `pulumi.com` provides is *concurrency control*. The service will allow
```

Comment:
I wonder if instead of saying "One of the services that `pulumi.com` provides ..." we should say something like 
"One of the services that the [Pulumi Service]({{< relref "/docs/intro/pulumi-service/" >}}) provides ..."